internal/proxy/impl.go

```diff
@@ -1630,6 +1630,142 @@ func (node *Proxy) Query(ctx context.Context, request *milvuspb.QueryRequest) (*
 	}, nil
 }
 
+func (node *Proxy) CreateAlias(ctx context.Context, request *milvuspb.CreateAliasRequest) (*commonpb.Status, error) {
+	if !node.checkHealthy() {
+		return unhealthyStatus(), nil
+	}
+	cct := &CreateAliasTask{
```

Comment:
The "cct" is abbreviation of "Create Collection Task". We use abbreviation for the task object. This is not a rule, but just keep the habit.
The CreateAliasTask should called "cat"
The DropAliasTask should called "dat"
The AlterAliasTask should called "aat"

Reply:
Got it. Updated.